features/FEATURE_BLE/ble/Gap.h

```diff
@@ -2447,6 +2455,13 @@ class Gap {
      */
     DisconnectionEventCallbackChain_t disconnectionCallChain;
 
+    /**
+     * Register a callback handling connection events to be used internally and serviced first.
+     *
+     * @param[in] callback Event handler being registered.
+     */
+    virtual void onConnectionPrivate(ConnectionEventCallback_t callback) { }
```

Comment:
This is not required and should be removed.

Reply:
thanks, made redundant by the connection event handler, removing